server/server.js

```diff
@@ -26,19 +27,23 @@ if (!isDev && cluster.isMaster) {
 } else {
   const app = express();
 
-  mongoose.connect(process.env.MONGODB_URI, { useNewUrlParser: true, useUnifiedTopology: true });
+  mongoose.connect(process.env.MONGODB_URI, {
+    useNewUrlParser: true,
+    useUnifiedTopology: true,
+  });
 
   // Priority serve any static files.
   app.use(express.static(path.resolve(__dirname, 'build')));
   app.use(express.json());
 
   app.use('/rpa-framework', rpaFrameworkRouter);
   app.use('/ssot', ssotRouter);
+  app.use('/robot', robotRouter);
```

Comment:
As stated above, I would rather have the route called something else, as robot is somewhat of a meta name in our software. We should specify more precise through those routes and their naming

Reply:
✅ I changed it to robot-execution-jobs